app/src/main/java/com/example/androidapplicationtask/FlashesRecyclerAdapter.kt

```diff
@@ -7,23 +7,22 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.Toast
+import androidx.databinding.DataBindingUtil
 import androidx.recyclerview.widget.RecyclerView
+import com.example.androidapplicationtask.databinding.FlashesListViewBinding
 import com.squareup.picasso.Picasso
 
 class FlashesRecyclerAdapter(val flashes: List<Post>, val context: Context) :
     RecyclerView.Adapter<FlashesRecyclerAdapter.FlahsesViewHolder>() {
     var url = "https://islam360.app/islam360/posts/image/urdu/"
```

Comment:
if there is no usage for this url kindly remove it

Reply:
done